### ISSUE TITLE
Fix integration test framework

### ISSUE THREAD

**Issue by dmage** [openshift/image-registry#52]
With the latest version of the origin container we get the authorization error in the integration tests.

> OpenShift access denied: User "admin" cannot get imagestreams/layers.image.openshift.io in project "namespace"

git bisect said that openshift/origin@b81b982406f5b9a7dbba2b3b0ff7edd702356408 is the first bad commit
```
commit b81b982406f5b9a7dbba2b3b0ff7edd702356408
Author: David Eads <deads@redhat.com>
Date:   Wed Jan 3 15:41:10 2018 -0500

    use cluster role aggregation for admin, edit, and view

:040000 040000 5d24ce1704026fa9c922a5e7c3c0bebb07e15e06 533af19293f56511d24d88b0a75c14528211cee3 M      pkg
:040000 040000 c8909287a9a6f4c5a6cc0bfafd40a890c092429c 9a7cb8beae0222f9d4ce2db4f18bf6f87d7d83e3 M      test
```

**Comment by dmage** [comment]
https://github.com/openshift/image-registry/blob/4e128e42af9068986de54f1ac152ef250e0c0e6f/pkg/testframework/project.go#L27-L39

It works with origin from 3.6, 3.7, 3.8, and 3.9 till b81b9824.

**Comment by dmage** [comment]
Adding time.Sleep(10 * time.Second) after project creation helps... :(

**Comment by dmage** [comment]
kubeClient.RbacV1() isn't available in our code: `type *kubernetes.Clientset has no field or method RbacV1`

**Comment by deads2k** [comment]
Are you running a complete master or just the API server.  You need the controllers too for a functional system.

I would suggest trying to work against something like `oc cluster up --version=the-right-one`.  Integration tests end up needing to start complete masters and even that is going to change in 3.10.

**Comment by dmage** [comment]
Fixed by #60.
